katalog/dual-nginx/patch/rbac.yml

```diff
@@ -5,3 +5,6 @@
 - op: "replace"
   path: "/rules/1/resourceNames"
   value: ["ingress-controller-leader-external", "ingress-controller-leader-internal"]
+- op: "replace"
+  path: "/rules/6/resourceNames"
+  value: ["ingress-controller-leader-external", "ingress-controller-leader-internal"]
```

Comment:
Do we need two patches? @angelbarrera92 ?

I think we need only the patch on the `/rules/6/resourceNames`